modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -464,25 +464,37 @@ async function extractSlots(input: PredictStep, predictors: Predictors): Promise
 }
 
 function MapStepToOutput(step: PredictStep, startTime: number): PredictOutput {
+  const entitiesMapper = (e?: EntityExtractionResult) => {
+    if (!e) {
+      return e
+    }
```

Comment:
Feels a bit weird, if you want it to be really 'null', then you can `return eval('null')`

Reply:
I don't want it to be null, I just don't want to change null to undefined.

I also don't really care about this part of the code as don't think any user will eventually test something like this:

```
if (event.nlu.slots[0].entity === null) {
    doSomething()
} else if (event.nlu.slots[0].entity === undefined) {
    doSomethingElse()
}

``` 

I'll just return undefined if e is falsy.

Reply:
talked to @Eff and we chose that entity will `null` if falsy.